posthog/src/main/java/com/posthog/internal/PostHogPreferences.kt

```diff
@@ -33,7 +33,7 @@ public interface PostHogPreferences {
         internal const val OPT_OUT = "opt-out"
         internal const val FEATURE_FLAGS = "featureFlags"
         internal const val FEATURE_FLAGS_PAYLOAD = "featureFlagsPayload"
-        internal const val SESSION_REPLAY = "sessionReplay"
+        public const val SESSION_REPLAY: String = "sessionReplay"
```

Comment:
rollback when https://github.com/PostHog/posthog-android/pull/199/files#r1819028697 is addressed most likely